source/administration/config-settings.rst

```diff
@@ -2042,9 +2042,23 @@ This filter uses the permissions of the **Bind Username** account to execute the
 | This feature's ``config.json`` setting is ``"UserFilter": ""`` with string input.                                                                                    |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Guest Filter
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+(Optional) Enter an AD/LDAP Filter to use when searching for external users who have Guest Access to Mattermost. Only the objects selected by the query will be accessible to Mattermost.
+
+See the `Guest Accounts documentation <https://docs.mattermost.com/deployment/guest-accounts.html>`__ for more information
+(available in Enterprise Edition E20 and higher).
+
+This filter is defaulted to ``(objectClass=guests)``.
```

Comment:
Default is blank or empty.

Reply:
So we'd say "The filter is defaulted to blank" or "the filter default is blank"? I guess it's because it's optional which I didn't consider - I used the UI text as a guideline so that's good to know. Thanks!

Reply:
I think I have a better idea of what you mean - the filter default is blank (the UI text is just a guideline) and values have to be explicitly added if being used?

Reply:
Have edited this.